src/catalog/src/builtin.rs

```diff
@@ -189,6 +189,12 @@ pub struct BuiltinIndex {
     pub is_retained_metrics_object: bool,
 }
 
+impl BuiltinIndex {
+    pub fn create_sql(&self) -> String {
+        format!("CREATE INDEX {}\nIN {}", self.name, self.sql)
```

Comment:
I think we should remove the `IN` from this method and keep it in the `sql` field.

Reply:
Sounds good -- the template will look like this then:

```rust
format!("CREATE INDEX {}\n{}", self.name, self.sql)

```

Reply:
^ That is to preserve the old formatting style